dev/com.ibm.ws.jca_fat_derbyra/test-resourceadapters/fvt-resourceadapter/src/fat/derbyra/resourceadapter/DerbyManagedConnectionFactory.java

```diff
@@ -141,8 +141,13 @@ public void setPassword(String password) {
         this.password = password;
     }
 
-    public void setQmid(String qmid) {
-        this.qmid = qmid;
+    public void setQmid(String qmid) throws ResourceException {
```

Comment:
I'm pretty sure there was a TODO in this test bucket that was the intended way of validating that QMID support had been added.  Why is this code being changed instead of following the TODO comment?

Reply:
I was attempting to fail the test if we did not get our expected qmid.   Still working on what would be best for a final automated test.   We need to check the value someplace to make sure the qmid that was used for the connection is the one used during recovery, if they do not match, fail the test.